services/provider/server/server.go

```diff
@@ -690,13 +705,12 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 			})
 
 		case "CephBlockPoolRadosNamespace":
-
 			rns := &rookCephv1.CephBlockPoolRadosNamespace{}
 			err = s.client.Get(ctx, types.NamespacedName{Name: cephRes.Name, Namespace: s.namespace}, rns)
 			if err != nil {
 				return nil, status.Errorf(codes.Internal, "failed to get %s CephBlockPoolRadosNamespace. %v", cephRes.Name, err)
 			}
-
+			mirroringEnabled := len(rns.Spec.Mirroring.Mode) > 0
```

Comment:
Could you change it to If rns.Spec.Mirroring is not nil then mirroring is enabled

Reply:
I think this is more precise check (it checks both)